toptal/picasso
# feat(tooltip): add compact size for tooltip component

[FX-1137]

### Description

- Add `compact` prop to `Tooltip` component for small size.
- Update `Slider` examples to show a compact version.

### How to test

- Verify `Tooltip` [example](https://picasso.toptal.net/fx-1137-add-small-variant-to-tooltip/?path=/story/overlays-folder--tooltip#compact) with `compact` prop
- Verify `Slider` displayed with a [compact tooltip](https://picasso.toptal.net/fx-1137-add-small-variant-to-tooltip/?path=/story/components-folder--slider#tooltip)

### Screenshots

| `Tooltip` with `compact` prop                                 | Slider with `compact` prop                                  |
| --------------------------------------- | --------------------------------------- |
| <img width="114" alt="Screen Shot 2020-09-01 at 10 12 37" src="https://user-images.githubusercontent.com/4870595/91855352-7053ff00-ec3b-11ea-869b-552a6dadd426.png"> | <img width="540" alt="Screen Shot 2020-09-01 at 10 24 59" src="https://user-images.githubusercontent.com/4870595/91856546-15bba280-ec3d-11ea-8d9a-cee40fa7e1bc.png"> |

### Review


<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run  whole pipeline
- `@toptal-bot run danger` - Danger checks
- `@toptal-bot run lint` - Run linter
- `@toptal-bot run test` - Run jest
- `@toptal-bot run build` - Check build
- `@toptal-bot run test:visual` or `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation


</details>


[FX-1137]: https://toptal-core.atlassian.net/browse/FX-1137
